howto/install-appliance/aws.md

```diff
@@ -0,0 +1,136 @@
+
+In addition, make sure you have the following accounts:
+
+* An Ubuntu SSO account. If you don't have one yet, create it [here](https://login.ubuntu.com).
+* An AWS account that you use to buy a subscription to the Anbox Cloud Appliance.
+  [note type="information" status="Note"]The quota for your AWS account must be sufficient for the instance types that you plan to use.[/note]
+
+## Install the appliance
+
+Complete the following steps to subscribe to the Anbox Cloud Appliance offering, get access to the required instances and configure them correctly.
+
+### 1. Start the launch wizard
+
+Open the [Amazon EC2 console](https://console.aws.amazon.com/ec2/) and log in.
+
+On the EC2 dashboard, click **Launch Instance** to start the Launch Instance Wizard.
+
+![Start the Launch Instance Wizard|690x451](https://assets.ubuntu.com/v1/17073a3d-install_appliance_launch-wizard.png)
+
+### 2. Select the AMI
+
+To select the Amazon Machine Image (AMI), type "Anbox Cloud" in the search field of the **Application and OS Images** section.
+
+![Search for the Anbox Cloud Appliance AMI|690x451](../../images/install_appliance_search-ami.png)
+
+Choose either the Arm variant or the x86 variant and click **Select**.
+
+![Select the Amazon Machine Image (AMI)|690x451](../../images/install_appliance_select-ami.png)
+
+You will be presented with the pricing information. Click **Continue** to confirm.
+
+### 3. Choose an instance type
+
+AWS offers various instance types. The Anbox Cloud Appliance images are supported for a subset of the available instance types only.
+
+In the **Instance type** section, select the instance type that is most suitable for what you're planning to do. For example, if you just want to try out the Anbox Cloud Appliance, an instance type with GPU support and limited CPU and memory is sufficient. See the [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734#appliance) for the minimum hardware requirements.
+
+![Choose an instance type|690x451](../../images/install_appliance_instance-type.png)
+
+In this example, we picked `g4dn.2xlarge`, which provides 8 vCPUs, 32 GB of memory and a single NVIDIA Tesla T4 GPU.
+
+### 4. Select a key pair
+
+In the **Key pair (login)** section, choose an existing key pair or create one if you don't have one yet. Make sure to save the private key in a secure location.
+
+![Choose or create a key pair|690x451](../../images/install_appliance_key-pair.png)
+
+### 5. Configure the network
+
+You do not need to customise any of the settings in the **Network settings** section, but you can fine-tune things. For example, you might want to put the instance onto a different VPC or subnet.
+
+To allow external access, several ports in the security group attached to the AWS instance must be open. The AMI already comes with the required configuration, so you don't need to do any changes. For reference, all required ports are documented [here](https://discourse.ubuntu.com/t/requirements/17734).
```

Comment:
May we add a note saying that all ports need to be opened but access can be limited to particular source IPs/subnets?